ChangeLog.md

```diff
@@ -18,8 +18,11 @@ to browse the changes between the tags.
 
 See docs/process.md for more on how version tagging works.
 
-3.1.22 (in development)
+3.1.23 (in development)
 -----------------------
+
+3.1.22 - 09/18/22
+-----------------
```

Comment:
I don't see the changelog entry about C++17 here, is this branch behind?

Reply:
Oops, fixed.